api/api.py

```diff
@@ -334,6 +353,52 @@ def historical(self, request, pk=None):
     def teste(self, request, pk=None):
         return Response("ola teste")
 
+    @action(detail=False)
+
+    def expense_revenue(self, request, pk=None):
+        """
+            Lista de expenses e revenues ordenada por ano
+        """
+        params = ExpensePerCountryIdSerializer(data=request.GET)
+        params.is_valid(raise_exception=True)
+        country = params.validated_data.get('country_id')
+        budget_summary = BudgetSummary.objects.filter(budget__country=country).order_by('budget__year')
+        list = []
+        for budget_summary in budget_summary:
+            if not (budget_summary.expense_functional_budget is None):
```

Comment:
Logic can be improved with Python: `if not (budget_summary.expense_functional_budget is None):` => `if budget_summary.expense_functional_budget:`